alpha/apps/kaltura/lib/batch2/kFlowManager.php

```diff
@@ -586,7 +586,12 @@ public function shouldConsumeChangedEvent(BaseObject $object, array $modifiedCol
 			{
 				return true;
 			}
-			
+
+		if ($object instanceof LiveEntry)
```

Comment:
Why not add this condition:
$object->isCustomDataModified(LiveEntry::CUSTOM_DATA_RECORDING_STATUS)
here as well you are doing it only in "objectChanged function"